Gemfile

```diff
@@ -15,14 +15,19 @@ gem "tzinfo-data", platforms: %i[ mingw mswin x64_mingw jruby ]
 gem "bootsnap", require: false
 gem 'bootstrap', '~> 5.3.1'
 gem 'simple_form'
+gem 'rails-controller-testing'
```

Comment:
sort the gem names alphabetically, and only use either single or double quotes. We prefer single quotes.